polkadotjs-examples/src/examples/creditcoin-eventsSubscribe.ts

```diff
@@ -0,0 +1,22 @@
+const { ApiPromise, WsProvider } = require('@polkadot/api');
+
+async function main() {
+    const wsProvider = new WsProvider('ws://127.0.0.1:9944');
+    const api = await ApiPromise.create({
+        provider: wsProvider,
+        rpc: {
+            creditcoin: {
+                eventsSubscribe: {
+                    description: 'Subscribe to events',
+                    params: [],
+                    type: 'Void'
+                }
+            }
+        }
+    });
+    await api.isReady;
+    await api.rpc.creditcoin.eventsSubscribe();
+}
+
+
+main().catch(console.error).finally(() => process.exit());
```

Comment:
@dlebee this example looks like it is very close but I get an error about the unknown Void type.

```bash
$ npx ts-node ./src/examples/creditcoin-eventsSubscribe.ts 
2022-03-28 10:56:27        API/INIT: RPC methods not decorated: creditcoin_eventsUnsubscribe, creditcoin_getEvents
2022-03-28 10:56:27        REGISTRY: Unable to resolve type Void, it will fail on construction
2022-03-28 10:56:27        RPC-CORE: eventsSubscribe(): Void:: createType(Void):: DoNotConstruct: Cannot construct unknown type Void
Error: createType(Void):: DoNotConstruct: Cannot construct unknown type Void
    at createTypeUnsafe (/home/ubuntu/creditcoin-substrate/polkadotjs-examples/node_modules/@polkadot/types-create/create/type.cjs:73:18)
    at TypeRegistry.createTypeUnsafe (/home/ubuntu/creditcoin-substrate/polkadotjs-examples/node_modules/@polkadot/types/create/registry.cjs:323:46)
    at RpcCore._formatOutput (/home/ubuntu/creditcoin-substrate/polkadotjs-examples/node_modules/@polkadot/rpc-core/bundle.cjs:413:21)
    at RpcCore._formatResult (/home/ubuntu/creditcoin-substrate/polkadotjs-examples/node_modules/@polkadot/rpc-core/bundle.cjs:222:27)
    at callWithRegistry (/home/ubuntu/creditcoin-substrate/polkadotjs-examples/node_modules/@polkadot/rpc-core/bundle.cjs:250:19)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Do you have an idea where it is defined? I tried grepping it but there are too many things which include "void" in their names. Will keep looking though.